src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/stats/MflStatSpec.groovy

```diff
@@ -264,7 +275,7 @@ class MflStatSpec extends HealthCheckSpecification {
         when: "Src switch is only left with 1 management controller (no stats controllers)"
         def regionToStay = srcSwitch.regions.find { flHelper.getFlByRegion(it).mode == RW }
         def blockData = lockKeeper.knockoutSwitch(srcSwitch, srcSwitch.regions - regionToStay)
-        switchHelper.waitForSwitchFlConnection(srcSwitch, false, flHelper.getFlsByRegions(srcSwitch.regions - regionToStay))
+        assert northboundV2.getSwitchConnections(srcSwitch.dpId).connections*.regionName == [regionToStay]
```

Comment:
Do you need to check connection mode too?

Reply:
Actually, I don't think so,
but I've added this verification, just in case.